code/src/App.js

```diff
@@ -1,9 +1,116 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ThoughtsList from 'components/ThoughtsList';
+import ThoughtForm from 'components/ThoughtForm';
+
 
 export const App = () => {
```

Comment:
Maybe make a wrapping component that can be mounted in the App.js to keep the App as clean as possible?